docs/docs/tutorial.md

```diff
@@ -395,6 +395,7 @@ var CommentBox = React.createClass({
     $.ajax({
       url: this.props.url,
       dataType: 'json',
+      cache: false,
```

Comment:
At the beginning of this code block (and the others), we have some numbers in curly braces. We use this to highlight lines (which you can see on the website, and [more info](http://zpao.com/posts/adding-line-highlights-to-markdown-code-fences/)).

You're adding a line right in the middle of this range and probably outside the ranges in other code blocks. We need to make sure they're all update appropriately.